### ISSUE TITLE
[A11Y][List item] Leading/Trailing elements have a wrong accessible name

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#927]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 17 / Samsung S25

### OS Version

iOS 26 / Android 16

### App version

3.0.0 (273)

### Describe the issue

- iOS: With leading/trailing image, we hear "Label, information image, **Label**" : visible Label is read twice

- Android: With leading/trailing image, we hear "Label, information image, **Image**" : remove the role "Image", useless here

- Both OS: With leading/trailing icon, we only here the visible Label, but we should here also the icon alternative

### Expected Behavior

Read visible Label only once
No image role
Read status icon at least for "Warning", "Error" and "Information" status 
Provide the same customisable example for icons, or may be with some contextual examples like warning or errors pictos.

**Comment by AhmedAmineZr** [comment]
hello @pya35 I make some correction in two other issues shared before this and I think that fix this issue . if possible we arrange a call to see if this ticket will be resolve with what I did before before I send new version for test for bug a11y of list item :)

**Comment by pya35** [comment]
@AhmedAmineZr After tests on build 729:

- **Fixed** iOS: With leading/trailing image, we hear "Label, information image, Label" : visible Label is read twice

- **Fixed** Android: With leading/trailing image, we hear "Label, information image, Image" : remove the role "Image", useless here

- **Not Fixed** Both OS: Leading and Trailing icons should have an alternative (whatever the status) that could be hidden if needed

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (732)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 927
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by AhmedAmineZr** [comment]
[AhmedAmineZr](https://github.com/AhmedAmineZr)
[1mo ago](https://github.com/Orange-OpenSource/ouds-flutter/issues/54#issuecomment-5308375136)
Member
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (732)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 927
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed
